app/services/alerts/adapters/analytics_adapter.rb

```diff
@@ -20,6 +20,16 @@ def benchmark_dates
         analysis_object.benchmark_dates(@analysis_date)
       end
 
+      def has_structured_content?
```

Comment:
Should we add these methods to the system adapter too?

Reply:
Yes, I think we should